src/components/cards/cardEmptyState/cardEmptyState.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { CardEmptyState } from '.';
+
+const meta: Meta<typeof CardEmptyState> = {
+    title: 'components/Cards/CardEmptyState',
+    component: CardEmptyState,
+    tags: ['autodocs'],
+    parameters: {
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/file/jfKRr1V9evJUp1uBeyP3Zz/v1.0.0?type=design&node-id=10055-28606&mode=design&t=dehPZplRn0YEdOuB-4',
+        },
+    },
+};
+
+type Story = StoryObj<typeof CardEmptyState>;
+/**
+ * Default EmptyStateCard component with minimum props.
+ *
+ * **Note:** see <EmptyState /> for more details on implementation of layout.
+ */
+export const Default: Story = {
+    args: {
+        heading: 'Heading',
+        objectIllustration: { object: 'LIGHTBULB' },
+    },
+};
+
```

Comment:
Technically an illustration isn't required, but I think with just a heading it's going be more confusing in the documentation as to what this component is/can do in relationship to EmptyState.